### ISSUE TITLE
Due date: August 10

### ISSUE THREAD

**Issue by creatornader** [w3c-ccg/nist-dig-comments#2]
This due date is fast approaching so we should try to get our response turned around as soon as possible.

I have emailed NIST requesting an extension on this deadline and CC'd the other work item owners @wyc @ChristopherA @kenhuangus. 

I will update this issue if/when I get a response.

**Comment by creatornader** [comment]
We received a response to our request.

From David M. Temoshok, <david.temoshok@nist.gov>:

> Thank you for your inquiry. Input, feedback and comments submitted in response to the NIST June 8, 2020 Request for Comments on Special Publication 800-63-3 Digital Identity Guidelines must be in writing to be considered. NIST may consider comments that are submitted subsequent to the August 10, 2020 comment period closing date, but we cannot ensure that comments submitted after the comment period closure date will have the same consideration.
> 
> Please advise if you have further questions or request additional information.
> 
> Thank you,
> 
> David Temoshok
> Senior Policy Advisor
> Applied Cybersecurity
> NIST IT Laboratory

Essentially, we can submit our commentary after the due date, but no guarantee it gets the "same consideration."

We have a decision to make. The way I see it, there are two reasonable options.

Proposal: Delay our submission and set a deadline let's say 2 weeks from now (around August 24rd), or try to get something together by tomorrow, knowing most people in the community wont get to weigh in.

Personally, I'd rather submit something a tad late that we're all happy with than rush to submit something to meet the deadline. We should decide ASAP in either case. Thoughts? @wyc @kenhuangus @ChristopherA

**Comment by kenhuangus** [comment]
We can submit two comments we have now and follow up with additional
comments.

Thanks

Ken

**Comment by creatornader** [comment]
We also have a live link to the site which will get updated as we add more comments: https://w3c-ccg.github.io/nist-dig-comments/

**Comment by wyc** [comment]
I'll be adding some from the original issue in a few mins

**Comment by wyc** [comment]
Agreed with @kenhuangus that we can submit our draft of comments now and then also again after corrections from the community. The goal here is to open discussion and dialog. We don't need to get everything 100% correct the first time, albeit we should have a good stab at it. That's why they have a draft as well.

**Comment by wyc** [comment]
Submitted. I wrote this:

> Hi David, you can find our community group's comments here:
> 
> https://w3c-ccg.github.io/nist-dig-comments/
> 
> I want to be clear that this document is not an official W3C position, as restated in the document. Rather, it is a collection of community responses that have been consolidated and slightly pruned by the editors. We received input from a wide variety of parties, including standards authors, industry, and the general public. Therefore, we hope you'll consider it as the beginning point of a larger discussion instead of any definitive stance or position with respect to the decentralized community ecosystem, and especially regarding the W3C standards including and related to Verifiable Credentials and Decentralized Identifiers. 
> 
> Because it is a living document, I have also attached an HTML file of its current version for your reference.
> 
> We worked on a pretty short timeline, so we believe the community will have additional comments and discussion for this document set. We editors will therefore make updates to the website above as we receive community input. Would it be helpful if we notified you after the consolidation of major changes?
> 
> Best,
> Wayne Chang
> Chair, W3C Credentials Community Group
>
